src/common/snippets/include/snippets/pass/mlp_seq_tokenization.hpp

```diff
@@ -32,7 +33,31 @@ namespace ov::snippets::pass {
 class TokenizeMLPSeqSnippets : public ov::pass::MatcherPass {
 public:
     OPENVINO_MATCHER_PASS_RTTI("snippets::pass::TokenizeMLPSeqSnippets");
-    explicit TokenizeMLPSeqSnippets(const SnippetsTokenization::Config& config);
+
+    /**
+     * @interface Config
+     * @brief Configuration for TokenizeMLPSeqSnippets pass
+     * @ingroup snippets
+     */
+    struct Config : public TokenizationConfig {
+        using CanBeFusedAsPostOpPred = std::function<bool(const std::shared_ptr<const ov::op::v0::MatMul>&,
+                                                          const std::shared_ptr<const ov::Node>&)>;
+
+        explicit Config(const TokenizationConfig& tokenization_config,
+                        CanBeFusedAsPostOpPred can_be_fused_as_postop = nullptr)
+            : TokenizationConfig(tokenization_config),
+              m_can_be_fused_as_postop(std::move(can_be_fused_as_postop)) {}
+
+        [[nodiscard]] const CanBeFusedAsPostOpPred& get_can_be_fused_as_postop() const {
```

Comment:
Shall we assert that pointer is not null anywhere? Or it should be asserted from outside (on function user side)?

In my opinion, we can even have a default value for it, like `[](...) {return false;}`

Reply:
I introduced default predicate as you suggested, and added an assert here. Also, I had to rearrange code in transformation pipeline because of these changes, please take a look